tests/bench/peer_card_bench.py

```
@@ -0,0 +1,433 @@
+    @honcho_llm_call(
+        provider=cast(Any, resolved_provider),
+        model=candidate.model,
+        track_name="Peer Card Call",
+        response_model=PeerCardQuery,
+        json_mode=True,
+        max_tokens=5000,
+        reasoning_effort="minimal",
+        enable_retry=True,
+        retry_attempts=1,  # unstructured output means we shouldn't need to retry, 1 just in case
+    )
+    async def call(old_peer_card: list[str] | None, new_observations: list[str]) -> Any:
```

Ruff (0.12.2)

Dynamically typed expressions (typing.Any) are disallowed in `call`

(ANN401)